packages/mainsail/source/helpers/parse-units.test.ts

```diff
@@ -0,0 +1,16 @@
+import { parseUnits } from "./parse-units";
+
+describe("parseUnits", async ({ assert, it, loader }) => {
+	it("should parse the value to wei", () => {
+		assert.equal(parseUnits(1, "wei"), "1");
+		assert.equal(parseUnits(1, "gwei"), "1000000000");
+		assert.equal(parseUnits(1, "ark"), "100000000");
```

Comment:
shouldn't this be 1 to 1e18?

```suggestion
		assert.equal(parseUnits(1, "ark"), "1000000000000000000");
```